js/src/common/states/AlertState.js

```diff
@@ -0,0 +1,6 @@
+export default class AlertState {
+  constructor(props = {}, key = Date.now()) {
```

Comment:
How safe is it to use the date for a unique identifier ? Isn't there a risk of collision if an extension opens two alerts with two consecutive lines of code ?

Wouldn't it be safer to use an incremental ID (the counter value could be in the manager state), or one of the random methods of javascript ?